### PR TITLE
chore: token list domain

### PR DESCRIPTION
**Pull request by hanszhoou**
[MASK-482](https://mask.atlassian.net/browse/MASK-482)

### CODE REVIEW

File: packages/mask/src/extension/background-script/EthereumServices/tokenList.ts [hunk 1]
```diff
@@ -102,7 +102,7 @@ async function fetchERC20TokensFromTokenList(urls: string[], chainId = ChainId.M
         }
 
         const tokens = await fetchCommonERC20TokensFromTokenList(x, chainId)
-        return { tokens, weight: x.includes('Mask-Token-List') ? 1 : 0 }
+        return { tokens, weight: x.includes('https://tokens.r2d2.to') ? 1 : 0 }
```
Comment (guanbinrui) [inline packages/mask/src/extension/background-script/EthereumServices/tokenList.ts:105]:
```suggestion
        return { tokens, weight: x.startsWith('https://tokens.r2d2.to') ? 1 : 0 }
```

File: packages/web3-constants/evm/ethereum.json [hunk 2]
```diff
@@ -1,30 +1,17 @@
...
-        "Mumbai": ["https://raw.githubusercontent.com/DimensionDev/Mask-Token-List/gh-pages/latest/80001/tokens.json"],
-        "Arbitrum": [
-            "https://raw.githubusercontent.com/DimensionDev/Mask-Token-List/gh-pages/latest/42161/tokens.json"
-        ],
-        "Arbitrum_Rinkeby": [
-            "https://raw.githubusercontent.com/DimensionDev/Mask-Token-List/gh-pages/latest/421611/tokens.json"
-        ],
-        "xDai": ["https://raw.githubusercontent.com/DimensionDev/Mask-Token-List/gh-pages/latest/100/tokens.json"]
+        "BSC": ["https://tokens.r2d2.to/latest/56/tokens.json", "https://api.1inch.exchange/v3.0/56/tokens"],
+        "BSCT": ["https://tokens.r2d2.to/mask.json"],
```
Comment (guanbinrui) [inline packages/web3-constants/evm/ethereum.json:9]:
```suggestion
        "BSCT": ["https://tokens.r2d2.to/latest/97/tokens.json"],
```

**Review by Lanttcat — DISMISSED** [review]

File: packages/web3-constants/evm/ethereum.json [hunk 3]
```diff
@@ -1,30 +1,17 @@
 {
     "ERC20_TOKEN_LISTS": {
-        "Mainnet": [
-            "https://api.1inch.exchange/v3.0/1/tokens",
-            "https://raw.githubusercontent.com/DimensionDev/Mask-Token-List/gh-pages/latest/1/tokens.json"
-        ],
-        "Ropsten": ["https://raw.githubusercontent.com/DimensionDev/Mask-Token-List/gh-pages/latest/3/tokens.json"],
-        "Rinkeby": ["https://raw.githubusercontent.com/DimensionDev/Mask-Token-List/gh-pages/latest/4/tokens.json"],
+        "Mainnet": ["https://tokens.r2d2.to/latest/1/tokens.json", "https://api.1inch.exchange/v3.0/1/tokens"],
```
Comment (guanbinrui) [inline packages/web3-constants/evm/ethereum.json:3]:
```suggestion
        "Mainnet": ["https://tokens.r2d2.to/latest/1/tokens.json"],
```

File: packages/web3-constants/evm/ethereum.json [hunk 4]
```diff
@@ -1,30 +1,17 @@
...
-        "Arbitrum": [
-            "https://raw.githubusercontent.com/DimensionDev/Mask-Token-List/gh-pages/latest/42161/tokens.json"
-        ],
-        "Arbitrum_Rinkeby": [
-            "https://raw.githubusercontent.com/DimensionDev/Mask-Token-List/gh-pages/latest/421611/tokens.json"
-        ],
-        "xDai": ["https://raw.githubusercontent.com/DimensionDev/Mask-Token-List/gh-pages/latest/100/tokens.json"]
+        "BSC": ["https://tokens.r2d2.to/latest/56/tokens.json", "https://api.1inch.exchange/v3.0/56/tokens"],
+        "BSCT": ["https://tokens.r2d2.to/latest/97/tokens.json"],
+        "Matic": ["https://tokens.r2d2.to/latest/137/tokens.json", "https://api.1inch.exchange/v3.0/137/tokens"],
```
Comment (guanbinrui) [inline packages/web3-constants/evm/ethereum.json:10]:
```suggestion
        "BSC": ["https://tokens.r2d2.to/latest/56/tokens.json"],
        "BSCT": ["https://tokens.r2d2.to/latest/97/tokens.json"],
        "Matic": ["https://tokens.r2d2.to/latest/137/tokens.json"],
```

**Comment by guanbinrui** [comment]
Blocked by https://github.com/DimensionDev/Mask-Token-List/pull/43.

**Review by Jack-Works — DISMISSED** [review]

**Review by Lanttcat — APPROVED** [review]

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**